pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -147,16 +169,45 @@ func (m *Multicluster) AddMemberCluster(clientset kubernetes.Interface, metadata
 		if valicationWebhookController != nil {
 			go valicationWebhookController.Start(stopCh)
 		}
+		//add config controller
+		remoteConfigController, err := m.createRemoteConfigController(clusterID, config, opts)
```

Comment:
Is this always enabled?  Should this only be enabled when centralIstio = true and when there is no workload going to be deployed to the istiod cluster?

Reply:
This has been guarded as  other remote controller via `if m.fetchCaRoot != nil {`